src/test/scala/com/cognite/sdk/scala/v1/AssetsTest.scala

```diff
@@ -64,6 +64,25 @@ class AssetsTest extends SdkTest with ReadBehaviours with WritableBehaviors {
     }
   )
 
+  it should "support updating by externalId" in {
```

Comment:
Seems this should be in a test trait and be done for all resources that support updating by `externalId` (which should be almost all of them?)